common/lib/capa/capa/templates/choicegroup.html

```diff
@@ -6,7 +7,7 @@
     ))
 %>
 <form class="choicegroup capa_inputtype" id="inputtype_${id}">
-    <fieldset ${describedby}>
+    <fieldset ${describedby | n, decode.utf8}>
```

Comment:
I'd be consistent and use:
```
${HTML(describedby)}
```
Also, if at all possible, try to rename anything that needs to be wrapped with `HTML()` with an `_html` suffix, like `describedby_html`.